packages/wordpress/components/blockStyles/blocks/separator.js

```diff
@@ -0,0 +1,9 @@
+import { createGlobalStyle } from 'styled-components';
+import { siteTheme } from '@irvingjs/styled/utils';
+
+export const SeparatorBlock = createGlobalStyle`
+  .wp-block-separator {
+    margin-left: 0;
+    width: ${siteTheme('blocks.separator.width', '95%')};
```

Comment:
I think the default width should be 100% (Defector is a little different than the norm because some blocks are contained to a max-width of 710px, while other are 100% of the container). Also, I wonder if this `margin-left` is needed. I feel like most sites would prefer the line to be centered, if it's not 100%.

I also think we should also add an option for color here:
```
border: 0;
border-top: ${siteTheme('blocks.separator.border', '1px solid black')};
```